README.md

```diff
@@ -0,0 +1,93 @@
+    - [Conformance](#conformance)
+- [How to build](#build)
+- [How to run](#run)
+- [How to contribute](#contribute)
+<!-- tocstop -->
+
+## <a name="what-is-enceladus"/>What is Enceladus?
+**Enceladus** is a **Dynamic Conformance Engine** which allows data from different formats to be standardized to parquet and conformed to group-accepted common reference (e.g. **DE** in one source system and **Deutschland** in another, can be conformed to **Germany**).
+
+The project is comprised of three main components:
+### <a name="menas"/>Menas
+This is the user-facing web client, used to **specify the standardization schema**, and **define the step required to conform** a dataset.  
+
+### <a name="standardization"/>Standardization
+This is a Spark job which reads in a dataset in any supported format and **produces a parquet dataset with the Menas-specified schema**. 
+
+### <a name="conformance"/>Conformance
+This is a Spark job which **applies the Menas-specified conformance rules to the standardized dataset**.
+
+## <a name="build"/>How to build
+#### Build requirements:
+ - **Maven 3.5.4+**
+ - **Java 8**
+
+Each module provides configuration file templates with reasonable default values.
+Make a copy of the `*.properties.template` and `*.conf.template` files in each module's `src/resources` directory removing the `.template` extension. 
+Ensure the properties there fit your environment.
+
+#### Build command:
+
+`mvn -DskipTests clean package`
+
+## <a name="run"/>How to run
+#### Menas requirements:
+- **Tomcat 8.5/9.0** installation
+- **MongoDB 4.0** installation
+- **HADOOP_CONF_DIR** environment variable, pointing to the location of your hadoop configuration
+
+#### Deploying Menas
+Simply copy the **menas.war** file produced whn building the project into Tomcat's webapps directory. 
+
+#### Standardization and Conformance requirements:
+- [**Spark 2.2.1-ABSA**](https://github.com/AbsaOSS/spark/tree/branch-2.2.1-ABSA) installation (this custom version can be [built like any other Spark version](https://spark.apache.org/docs/latest/building-spark.html))
+- **Hadoop 2.7** installation
+
+#### Running Standardization
+```
+<spark home>/spark-submit \
+--num-executors <num> \
+--executor-memory <num>G \
+--master yarn --deploy-mode <client/cluster> \
+--driver-cores <num> \
+--driver-memory <num>G \
+--conf "spark.driver.extraJavaOptions=-Dmenas.rest.uri=<menas api uri:port> -Dstandardized.hdfs.path=<path for standardised output>-{0}-{1}-{2}-{3} -Dspline.mongodb.url=<mongo url for spline> -Dspline.mongodb.name=<spline database name> -Dhdp.version=<hadoop version>" \
+--class za.co.absa.enceladus.standardization.StandardizationJob \
+<standardization-<build-version>.jar> \
+--dataset-name <Dataset Name> \
+--dataset-version <Dataset Version> \
+--report-date <Date> \
+--report-version <Data Run Version> \
+--raw-format <data format> \
+--row-tag <tag> \
+--header <true/false(applicable for CSV)> \
+--delimiter <csv delimiter (applicable for CSV)>
```

Comment:
Having both row-tag and header + delimiter here can be misleading. I propose to add a standalone section about file formats and options there.